pkgs/top-level/aliases.nix

```diff
@@ -1473,6 +1473,7 @@ mapAliases ({
   quake3game = throw "'quake3game' has been renamed to/replaced by 'ioquake3'"; # Converted to throw 2022-02-22
   quaternion-git = throw "quaternion-git has been removed in favor of the stable version 'quaternion'"; # Added 2020-04-09
   quilter = throw "quilter has been removed from nixpkgs, as it was unmaintained"; # Added 2021-08-03
+  qutebrowser-qt6 = throw "'qutebrowser-qt6' has been replaced by 'qutebrowser', since the the qt5 version has been removed"; # Added 2023-08-19
```

Comment:
Why not
```suggestion
  qutebrowser-qt6 = qutebrowser; # Added 2023-08-19
```
?

Reply:
I followed other Qt applications that did a similar migration.